packages/@aws-cdk/aws-certificatemanager/lib/dns-validated-certificate.ts

```diff
@@ -52,7 +53,8 @@ export interface DnsValidatedCertificateProps extends CertificateProps {
  * validated using DNS validation against the specified Route 53 hosted zone.
  *
  * @resource AWS::CertificateManager::Certificate
- * @experimental
+ * @deprecated use the `validation` prop with `CertificateValidation.fromDns()`
+ * on `Certificate`.
```

Comment:
Counter-argument -- this custom resource-based approach allows for cross-region cert creation, which the native CloudFormation support doesn't allow (AFAIK). It might be worth not deprecating it, and noting this use case in the README. What do you think?

Reply:
Good point. For information, it's in the CF roadmap https://github.com/aws-cloudformation/aws-cloudformation-coverage-roadmap/issues/523.

This also solves my question here https://github.com/aws/aws-cdk/pull/8552#issuecomment-644349690